resources/definitions/z-bolt_base.def.json

```diff
@@ -0,0 +1,279 @@
+{
+    "name": "Z-Bolt Base",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": 
+    {
+        "visible": false,
+        "author": "Z-Bolt",
+        "manufacturer": "Z-Bolt Co.",
+        "file_formats": "text/x-gcode",
+        "first_start_actions": ["MachineSettingsAction"],
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants": true,
+        "variants_name": "Toolhead",
+        "preferred_variant_name": "Direct E3DV6 0.40mm",
+        "machine_extruder_trains": { "0": "z-bolt_extruder_0" },
+        "exclude_materials": [
+            "ultimaker_abs_black",
```

Comment:
It's really not needed to exclude these material profiles! They are 2.85 mm, so they won't show up for this printer anyway.

Reply:
Removed Ultimaker material profiles from blacklist.